# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../koffice/gcc41.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- koffice-1.4.2/krita/core/kis_filter_registry.h	2005/05/27 09:18:23	418647
+++ koffice-1.4.2/krita/core/kis_filter_registry.h	2005/10/24 15:53:42	473737
@@ -26,7 +26,6 @@
 #include "kis_types.h"
 #include "kis_generic_registry.h"
 #include "kaction.h"
-#include "kis_filter.h"
 
 class QString;
 class KisView;
--- koffice-1.4.2/krita/core/kis_clipboard.h.orig	2006-04-02 12:47:07.000000000 +0200
+++ koffice-1.4.2/krita/core/kis_clipboard.h	2006-04-02 12:47:20.000000000 +0200
@@ -35,7 +35,7 @@ class KisClipboard : public QObject {
 
 public:
 
-	virtual KisClipboard::~KisClipboard();
+	virtual ~KisClipboard();
 
 	static KisClipboard* instance();
 
--- koffice-1.4.2/kspread/kspread_cell.h.orig	2006-04-02 15:52:37.000000000 +0200
+++ koffice-1.4.2/kspread/kspread_cell.h	2006-04-02 15:53:42.000000000 +0200
@@ -596,7 +596,7 @@ public:
      *
      * @return the cell that decides the format for the cell in question.
      */
-    KSpreadCell *KSpreadCell::ultimateObscuringCell() const;
+    KSpreadCell *ultimateObscuringCell() const;
 
     /**
      * @return the obscuring cell list (might be empty)
